.pipelines/templates/e2e-windows-template.yaml

```diff
@@ -95,6 +98,16 @@ jobs:
         export WINDOWS_E2E_IMAGE=${{ parameters.windowsImage }}
         export WINDOWS_DISTRO=${{ parameters.windowsDistro }}
         export WINDOWS_NODE_IMAGE_VERSION=${{ parameters.windowsNodeImageVersion }}
+        export CONFIG_GPU_DRIVER_IF_NEEDED=false
+        export GPU_SUFFIX=""
+        if [[ "${{ parameters.windowsGpuDriverSuffix }}" == "-grid" ]]; then
+          export WINDOWS_GPU_DRIVER_URL="https://download.microsoft.com/download/9/d/6/9d6f3611-ff0c-43bc-8958-fe7fb0ded78d/537.13_grid_win10_win11_server2019_server2022_dch_64bit_international_azure_swl.exe"
```

Comment:
use the driver binaries uploaded to https://acs-mirror.azureedge.net/aks/windows/gpudrivers/grid